rules/packages-up-to-date/rule.md

```diff
@@ -32,13 +32,13 @@ Npm packages often get new releases (adding new functionalities, fixing bugs or
 <!--endintro-->
 
 Running `npm outdated` returns an overview of your packages versions showing: 
-- The **current** version your project is using.
-- The **wanted** version i.e. the maximum version of the package that satisfies the semver range specified in package.json.
-- The **latest** version of the package.
+- The **current** version your project is using
+- The **wanted** version. E.g. The maximum version of the package that satisfies the semver range specified in package.json
```

Comment:
I think that this should still be i.e. as it is just giving a longer explanation of what "wanted" means, rather than an example